ironic/drivers/modules/drac/management.py

```diff
@@ -302,14 +315,182 @@ def set_boot_device(node, device, persistent=False):
 
 class DracRedfishManagement(redfish_management.RedfishManagement):
     """iDRAC Redfish interface for management-related actions.
-
-    Presently, this class entirely defers to its base class, a generic,
-    vendor-independent Redfish interface. Future resolution of Dell EMC-
-    specific incompatibilities and introduction of vendor value added
-    should be implemented by this class.
     """
-    pass
+    @METRICS.timer('DracRedfishManagement.clear_job_queue')
+    @base.clean_step(priority=0)
+    def clear_job_queue(self, task):
```

Comment:
I think we need to split this into 2 methods: clear_job_queue(), which returns None on success, and _clear_job_queue() which returns the results.